package-lock.json

```diff
@@ -30499,14 +30499,15 @@
       }
```

Comment:
The 'marked' package version has been updated in the package-lock.json file. This change should be automatically generated by running npm install. Please make sure that npm install was run after updating the package.json file.